### DISCUSSION TITLE
Fegefeuer High Rounds

### DISCUSSION THREAD

**Discussion by bignoobus** [nzp-team/nzportable#189] [Records and Speedruns]
go post ur high rounds for fege here foo

**Comment by Paindeluxe** [comment]
![fte-20230213220942-0](https://user-images.githubusercontent.com/104042185/218578681-c9b789aa-0899-4ba2-a5db-1b88dde17e25.png)


https://drive.google.com/file/d/1rUv2-483zUZug6AZHK9lOqtc-oBF8uEH/view?usp=sharing Place this shit into nzp then say playdemo Maniac_Plays_Fegefeuer(GONE_WRONG) in the console 

Or watch a very fast playback of it right [here ](https://youtu.be/ABpy1cKw5T0)(:

**Reply by Paindeluxe** [reply]
I was so close to reaching round 40 but hey there's prob another day to try it again unless...?

**Comment by Paindeluxe** [comment]
Goddamn was this boring I just decided to end game cuz I was basically sleeping I had a revive left
![fte-20230214200546-0](https://user-images.githubusercontent.com/104042185/218833585-d984c304-1c42-416e-9270-c0a0953e9059.png)
[Demo](
https://drive.google.com/file/d/14W_8gp2Nm3SH2hKUoO_YvYIN-2ctPfEt/view?usp=share_link)

**Reply by MotoLegacy** [reply]
Edited your comment to remove bad conduct phrase.

**Reply by Paindeluxe** [reply]
Haha get noobed on the both of you (:<<<<

![fte-20230314195917-0](https://user-images.githubusercontent.com/104042185/225128083-be7671fd-51d0-4b8b-82c8-4cceccd34d80.png)
![fte-20230314202528-0](https://user-images.githubusercontent.com/104042185/225128093-3b0c18bc-e200-48cd-ae8d-242388ae9c6d.png)
![fte-20230314202531-0](https://user-images.githubusercontent.com/104042185/225128098-be5c7c33-403e-4699-a7f3-432a909cebc3.png)
![fte-20230314211748-0](https://user-images.githubusercontent.com/104042185/225128103-fb6a2592-f53e-47bc-a3e2-7d58f9ef4aa8.png)
![fte-20230314211753-0](https://user-images.githubusercontent.com/104042185/225128113-498972a4-bea4-4152-b1fb-a251dd2a0d90.png)
![fte-20230314211824-0](https://user-images.githubusercontent.com/104042185/225128122-c98e8a25-0b83-4905-a4e1-d05b5562cc37.png)

Note this was ran on the previous version of Fegefeuer.

[DEMO](https://drive.google.com/file/d/1FfFwPEqxPCZHrDzVS78bnn7f6lJA7R_Y/view?usp=share_link)

**Comment by notrickzdumbo** [comment]
That was the shortest lived record lol
Died to the trap after downing rip. Could be pushed further of course.
![fte-20230314215721-0](https://user-images.githubusercontent.com/98246579/225185645-fdab618c-0f5f-49a3-b099-c2e0161cf2a7.png)
Played possibly one nightly behind as of posting.(I don't even remember if I updated or not)

**Comment by Paindeluxe** [comment]
I hate this
![fte-20230315170739-0](https://user-images.githubusercontent.com/104042185/225379553-30911e89-c95b-4794-890e-b15d4804ee14.png)
![fte-20230315170800-0](https://user-images.githubusercontent.com/104042185/225379605-4e91f0f8-fd7d-472d-8aca-aeb4f7e9ad61.png)

**Comment by toiletman74** [comment]
![image](https://user-images.githubusercontent.com/91436591/226241726-e02a827a-d7a5-4914-9bc8-e97c5ef8d0ae.png)
Killed the run with 3 revives left cause the strat I was using was SUPER boring

**Comment by bignoobus** [comment]
**_All of your WR are belong to me._**
took probably well over 2 hours 
### _**gg**_
![fte-20230501194540-0](https://user-images.githubusercontent.com/105007743/235511119-3275c96c-7183-43b7-b2eb-811544159041.png)

**Comment by SESHZombies** [comment]
Nazi Zombies: Portable : Fegefeuer Round 57 WR

https://www.youtube.com/watch?v=w1DaPl855hg

![Capture](https://github.com/user-attachments/assets/d27326aa-d0b6-41ea-b78d-6d037f262b9f)

**Reply by MotoLegacy** [reply]
Do you have issues with the in game stopwatch, or did you not know it was a feature?
`cl_stopwatch` is the cvar I think. a value of 1 just gets you the total time, 2 gets you total time + time in current round

**Reply by SESHZombies** [reply]
Didn't know it was a feature

**Comment by conbobcob** [comment]
hey did you know theres a glitch you can get infinite rounds just stand behind the tipped over deadshot machine and the zombies cant get you and make sure not to open both doors in the building
![Screenshot 2024-07-17 3 35 55 PM](https://github.com/user-attachments/assets/08fdc7f7-6f0d-454d-8699-f832fb21123d)

**Reply by Galladite27** [reply]
> hey did you know theres a glitch you can get infinite rounds just stand behind the tipped over deadshot machine and the zombies cant get you and make sure not to open both doors in the building

You can also stand on top of the machine using the small box on the other side of the machine, and the zombies will pathfind to stand in front of it, even avoiding other players on the ground. Then you can open both doors in the building.

![Picture](https://github.com/user-attachments/assets/c7c30ab1-19ed-4e8c-9829-e0388aa9175e)

**Reply by conbobcob** [reply]
ok good to know
